src/utils/managing.officer.corporate.ts

```diff
@@ -16,20 +17,26 @@ import {
 import { ManagingOfficerCorporateKey, ManagingOfficerCorporateKeys } from "../model/managing.officer.corporate.model";
 import {
   AddressKeys,
+  EntityNumberKey,
   HasSamePrincipalAddressKey,
   ID,
+  InputDateKeys,
   IsOnRegisterInCountryFormedInKey,
   PublicRegisterNameKey,
   RegistrationNumberKey
 } from "../model/data.types.model";
 import { PrincipalAddressKey, PrincipalAddressKeys, ServiceAddressKey, ServiceAddressKeys } from "../model/address.model";
+import { ResignedOnDateKey, ResignedOnDateKeys, StartDateKey, StartDateKeys } from "../model/date.model";
 
 export const getManagingOfficerCorporate = (req: Request, res: Response, backLinkUrl: string, templateName: string) => {
   logger.debugRequest(req, `${req.method} ${req.route.path}`);
 
+  const appData: ApplicationData = getApplicationData(req.session as Session);
+
   return res.render(templateName, {
     backLinkUrl: backLinkUrl,
-    templateName: templateName
+    templateName: templateName,
+    entity_number: appData[EntityNumberKey]
```

Comment:
Any impact on registration here? Looks to be a shared controller

Reply:
It shouldn't be, it just makes the `entity_number` available to the templates but its just used to drive whether to show the start_date and resigned_on fields or not

Reply:
And I've ran through the Registration journey to add New MOs and both for individual and corporate its all still as is!